waffle/compat.py

```diff
@@ -1,13 +1,3 @@
-import django
+from django.conf import settings
 
-__all__ = ['User', 'AUTH_USER_MODEL']
-
-# Django 1.5+ compatibility
-if django.VERSION >= (1, 5):
-    from django.contrib.auth import get_user_model
-    from django.conf import settings
-    User = get_user_model()
-    AUTH_USER_MODEL = settings.AUTH_USER_MODEL
-else:
-    from django.contrib.auth.models import User
-    AUTH_USER_MODEL = 'auth.User'
+AUTH_USER_MODEL = getattr(settings, 'AUTH_USER_MODEL', 'auth.User')
```

Comment:
It's always a string. settings.AUTH_USER_MODEL should reference to the module to be imported.